src/internal/github/client.go

```diff
@@ -11,14 +11,18 @@ import (
 	"github.com/shurcooL/githubv4"
 )
 
+const UserAgent = "OpenTofu Registry/1.0"
+
+// EnvAuthToken returns the GitHub token from the environment.
 func EnvAuthToken() (string, error) {
 	token := os.Getenv("GH_TOKEN")
 	if token == "" {
-		return "", fmt.Errorf("Expected $GH_TOKEN to be set, unable to authenticate with GitHub")
+		return "", fmt.Errorf("expected $GH_TOKEN to be set, unable to authenticate with GitHub")
```

Comment:
maybe `expected the GH_TOKEN environment variable to be set` to make it explicit?